### ISSUE TITLE
Metrics / Grafana setup for the Fluffy fleet.

### ISSUE THREAD

**Issue by kdeme** [status-im/infra-nimbus#92]
Since the launch of the Fluffy testnet/fleet (https://github.com/status-im/infra-nimbus/issues/87) quite some metrics have been added to Fluffy.
We are at the stage where it would be useful to be able to track these for our fleet.

I assume this means a setup of Prometheus and Grafana, similar to how it is done for nimbus-eth2.
I'd also like to have a read only public access to the Grafana dashboard, as exists for nimbus-eth2.

I have no idea how far these setups scale for a lot of nodes, so if doing this for 32 nodes in such dashboard is too much, it is fine to just select a sample of the nodes. Having a sample would be already sufficient to understand what is going on in the network.

I think we can use the same system with `instance` selector in the grafana dashboard?

There exists an Grafana dashboard with useful metrics for Fluffy here:
https://github.com/status-im/nimbus-eth1/blob/master/fluffy/grafana/fluffy_grafana_dashboard.json

It does not contain yet the `instance` variable. I can add that later.

**Comment by jakubgs** [comment]
Yeah, the dashboard works, but some parts don't look right without a `container` label being used to filter metrics:

![image](https://user-images.githubusercontent.com/2212681/173836884-6adf9ca8-2682-436a-8d8c-27f8c7883b0a.png)

https://metrics.status.im/d/iWQQPuPnkadsf/nimbus-fluffy-dashboard
